docs/v1/use-cases.md

```diff
@@ -1,96 +1,146 @@
-would allow these operators to participate in the existing service broker
-ecosystem and grow their user base accordingly.
+A user must register each service broker with the service catalog to advertise
+the services it offers to the catalog. After the broker has been registered
+with the catalog, the catalog makes a call to the service broker's `/v2/catalog`
+endpoint. The broker's returns a list of services offered by that broker. Each
+Service has a set of plans that differentiate the tiers of that service.
 
-## CF Service Broker `v2` API Use Cases
+### Updating a service broker
 
-Initially, the catalog should support the current [CF Service Broker
-API](https://docs.cloudfoundry.org/services/api.html) These are the use cases
-that the service catalog has to implement in order to use that API.
-
-### Managing service brokers
-
-1.  As user, I want to be able to register a broker with the Kubernetes service
-    catalog, so that the catalog is aware of the services that broker offers
-2.  As a user, I want to be able to update a registered broker so that the
-    catalog can maintain the most recent versions of services that broker offers
-3.  As a user, I want to be able to delete a broker from the catalog, so that I
-    can keep the catalog clean of brokers I no longer want to support
-
-#### Registering a service broker with the catalog
-
-An user must register each service broker with the service catalog to
-advertise the services it offers to the catalog.  After the broker has been
-registered with the catalog, the catalog makes a call to the service broker's
-`/v2/catalog` endpoint.  The broker's returns a list of services offered by
-that broker.  Each Service has a set of plans that differentiate the tiers of
-that service.
-
-#### Updating a service broker
-
-Broker authors make changes to the services their brokers offer.  To refresh the
+Broker authors make changes to the services their brokers offer. To refresh the
 services a broker offers, the catalog should re-list the `/v2/catalog` endpoint.
 The catalog should apply the result of re-listing the broker to its internal
 representation of that broker's services:
 
-1.  New service present in the re-list results are added
-2.  Existing services are updated if a diff is present
-3.  Existing services missing from the re-list are deleted
+1. New service present in the re-list results are added
+2. Existing services are updated if a diff is present
+3. Existing services missing from the re-list are deleted
 
 TODO: spell out various update scenarios and how they affect end-users
 
-#### Delete a service broker
+### Delete a service broker
 
-There must be a way to delete brokers from the catalog.  In Cloud Foundry, it is
-possible to delete a broker and leave orphaned service instances.  We should
+There must be a way to delete brokers from the catalog. In Cloud Foundry, it is
+possible to delete a broker and leave orphaned service instances. We should
```

Comment:
Hi @slack ;)

CF prevents removal of a broker when service instances of any service associated with the broker are present. I'll admit, the following UX would be nicer if the error was returned before asking for confirmation, but we like to keep our clients dumb and let the API do the validation.

```
$ cf delete-service-broker scoen-rs

Really delete the service-broker scoen-rs?> y
Deleting service broker scoen-rs as admin...
FAILED
Server error, status code: 400, error code: 270010, message: Can not remove brokers that have associated service instances: scoen-rs
```